sbnobj/Common/Reco/classes_def.xml

```diff
@@ -72,7 +72,9 @@
   <class name="art::Wrapper<art::Assns<float,recob::Shower,void>>" />
   <class name="art::Wrapper<art::Assns<recob::Shower,float,void>>" />
 
-  <class name="sbn::Stub" ClassVersion="12">
+  <class name="sbn::Stub" ClassVersion="14">
+   <version ClassVersion="14" checksum="3533111376"/>
+   <version ClassVersion="13" checksum="3160890792"/>
```

Comment:
Consider squashing the temporary version.
```suggestion
  <class name="sbn::Stub" ClassVersion="13">
   <version ClassVersion="13" checksum="3533111376"/>
```

Reply:
Did it!